src/components/lists/TaskList.vue

```diff
@@ -911,8 +915,13 @@ export default {
           this.formatDate(task.done_date),
           this.formatDate(task.last_comment_date)
         ]
-        if (!this.isAssets) {
-          const value = this.getEntity(task.entity.id).nb_frames
+        if (this.isShots) {
+          let value = 0
+          if (this.isShots && !this.isPaperProduction) {
+            value = this.getEntity(task.entity.id).nb_frames
+          } else {
+            value = task.nb_drawings || 0
+          }
```

Comment:
Duplidate condition `this.isShots`

```suggestion
        if (this.isShots) {
          const value = !this.isPaperProduction
            ? this.getEntity(task.entity.id).nb_frames
            : task.nb_drawings || 0
          line.splice(4, 0, value)
        }
```